test_scripts/Polices/External_UCS/External_Consent_OFF/028_ATF_Policies_External_Consent_OFF_same_notification_user_consent_allowed_externalConsentStatus_disallowed.lua

```diff
@@ -0,0 +1,134 @@
+--------------------------------------Requirement summary---------------------------------------------
+-- [Policies] External UCS: externalConsentStatus vs. consentedFunctions priority
+
+------------------------------------General Settings for Configuration--------------------------------
+require('user_modules/all_common_modules')
+local common_functions_external_consent = require('user_modules/ATF_Policies_External_Consent_common_functions')
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
```

Comment:
@maria-mih below lines are in all_common_modules.lua. We do not need to keep them here.
```
+local common_steps = require('user_modules/common_steps')
+local common_functions = require ('user_modules/common_functions')
```